src/ResourceManager/Insights/Commands.Insights/TransitionalClasses/MetricTrigger.cs

```diff
@@ -25,17 +25,29 @@ public class MetricTrigger : Monitor.Models.MetricTrigger
         /// <summary>
         /// Gets or sets the OperatorProperty of the Metric Trigger
         /// </summary>
-        public new ComparisonOperationType OperatorProperty { get; set; }
+        public new ComparisonOperationType OperatorProperty
+        {
+            get => (ComparisonOperationType)System.Enum.Parse(typeof(ComparisonOperationType), base.OperatorProperty.ToString());
+            set => base.OperatorProperty = (Monitor.Models.ComparisonOperationType)System.Enum.Parse(typeof(Monitor.Models.ComparisonOperationType), value.ToString());
```

Comment:
Expression bodied members for set/get won't work on our build system. Currently (AzureRM) builds against C# 6. This feature wasn't introduced until C# 7.